src/Integration/Binding/SolutionBindingOperation.cs

```diff
@@ -153,38 +150,8 @@ public void Prepare(CancellationToken token)
         }
 
         public bool CommitSolutionBinding()
-        {
```

Comment:
We used to support running an action after the main part of the "commit binding" had been done, which was only every used in "legacy connected mode".

Since we're auto-migrating settings to the new-new Connected Mode format, we don't need to support saving in this legacy format.